test/CMakeLists.txt

```diff
@@ -639,6 +651,22 @@ if(iodaconv_bufr_ENABLED)
                             gdas.t18z.1bmhs.tm00.15.7.filter_split.nc ${IODA_CONV_COMP_TOL_ZERO}
                     DEPENDS bufr2ioda.x )
 
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_1bamua2ioda
+                    TYPE    SCRIPT
+                    COMMAND ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                    ARGS    netcdf
+                            "${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_1bamua.yaml"
+                            gdas.t12z.1bamua.tm00.noaa-19.nc ${IODA_CONV_COMP_TOL_ZERO}
+                    DEPENDS bufr2ioda.x )
+
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_esamua2ioda
+                    TYPE    SCRIPT
+                    COMMAND ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                    ARGS    netcdf
+                            "${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_esamua.yaml"
+                            gdas.t12z.esamua.tm00.noaa-19.nc ${IODA_CONV_COMP_TOL_ZERO}
+                    DEPENDS bufr2ioda.x )
```

Comment:
@rmclaren There are two tests related to BUFR2IODA conversion in this PR:
test_iodaconv_bufr_1bamua2ioda and test_iodaconv_bufr_esamua2ioda

The test_iodaconv_bufr_1bamua2ioda takes 12 seconds to complete.  However, the test_iodaconv_bufr_esamua2ioda takes about 110 seconds to finish.

The size of the input BUFR file for these two tests are the following:
gdas.t12z.1bamua.tm00.bufr_d  --- 3241840 kb
gdas.t12z.esamua.tm00_bufr_d --- 2619528 kb

The run time for esamua is much longer despite the size of the input files are similar.  

The offline tests using the following:
/build/bin/bufr2ioda.x  bufr_ncep_1bamua.yaml
/build/bin/bufr2ioda.x  bufr_ncep_esamua.yaml

The offline tests take approximately the same length of  time (~15 seconds) to complete.